src/App.js

```diff
@@ -20,27 +20,32 @@ console.log(props, "PROPS TO YAS")
     <div className="App-header">
       <h1>Redux To-Do List</h1>
       <ul>
-        {props.todos.map((item, index) => 
-          <div key={index}>
-            {item.complete ? 
-              <i className="far fa-check-square status-icon"></i> : 
-              <i onClick={() => props.changeTodoStatus(item)} className="far fa-square status-icon"></i>}
-            <li>{item.task}</li>
-            <i 
-              onClick={() => props.deleteFromList(item)} 
-              className="delete far fa-times-circle" 
-              
-            ></i>
-          </div>
-          
+        {props.todos.length < 1 ? <div><p>Add some tasks below!! :D</p></div> : 
+          props.todos.map((item, index) => 
+            <div key={index}>
+              {item.complete ? 
+                <i onClick={() => props.changeTodoStatus(item)} className="far fa-check-square status-icon"></i> : 
+                <i onClick={() => props.changeTodoStatus(item)} className="far fa-square status-icon"></i>}
+              <li>{item.task}</li>
+              <i 
+                onClick={() => props.deleteFromList(item)} 
+                className="delete far fa-times-circle" 
+                
+              ></i>
+            </div>
         )}
       
       </ul>
       
-      <p>Enter Todo item:</p>
```

Comment:
when you use `<button>`, you'd want to wrap the whole `input` and `button` in a `<form>` in order for the submit to happen. Right now, you need to keep pressing the button to submit, but that would be not ideal in a real life situation.
So to do this, you wrap:
```html
<form onSubmit={}>
    <input...
    <button...
</form>

Reply:
Another thing to note, I would rename it to `handleSubmit`, to make it more consistent.

But code-wise, something to note about this function:
```javascript
  const submit = (text) => {
    props.addTodo(text);
    setText("");
  };
```
1. You don't need to pass `text` because it's already being referenced in the scope of this. So you can just use text without passing it
2. You should pass the `event` param to do `e.preventDefault()` You can read about it: https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault
3. That means when you put it on the `<form onSubmit={() => submit(text)}>` it can look like this `<form onSubmit={submit}>`

You want to avoid using inline arrow functions because it affects render